docs/apm-breaking.asciidoc

```diff
@@ -5,49 +5,147 @@
+
+[float]
+==== New {es} output
+
+APM Server has a new {es} output implementation with defaults that should be sufficient for
+most use cases; it should no longer be necessary to manually tune the output
+of APM Server.
+
+As a part of this change, the following settings have been removed:
+
+* `output.elasticsearch.bulk_max_size`
 * `output.elasticsearch.worker`
 * `queue.*`
-* `setup.*`
 
-The following configuration variables have been introduced:
+[float]
+==== New source map upload endpoint
+
+The source map upload endpoint has been removed from APM Server.
+Source maps should now be uploaded directly to {kib} instead.
+
+[float]
+==== Legacy Jaeger endpoints have been removed
+
+The legacy Jaeger gRPC and HTTP endpoints have been removed from APM Server.
+
+As a part of this change, the following settings have been removed:
+
+* `apm-server.jaeger`
+
+[float]
+==== All removed and changed settings
+
+Below is a list of all **removed settings** (in alphabetical order) for
+users upgrading a standalone (legacy) APM Server to {stack} version 8.0.
+
+[source,yml]
+----
+apm-server.data_streams.enabled
+apm-server.ilm.*
+apm-server.jaeger
+apm-server.register.ingest.pipeline.*
+apm-server.sampling.keep_unsampled
+output.elasticsearch.bulk_max_size
+output.elasticsearch.index
+output.elasticsearch.indices
+output.elasticsearch.pipeline
+output.elasticsearch.pipelines
+output.elasticsearch.worker
+queue.*
+setup.*
+----
+
+Below is a list of **renamed settings** (in alphabetical order) for
+users upgrading a standalone (legacy) APM Server to {stack} version 8.0.
+
+[source,yml]
+----
+previous setting                --> new setting
+
+apm-server.api_key              --> apm-server.auth.api_key
+apm-server.instrumentation      --> instrumentation
+apm-server.rum.allowed_service  --> apm-server.auth.anonymous.allow_service
+apm-server.rum.event_rate       --> apm-server.auth.anonymous.rate_limit
+apm-server.secret_token         --> apm-server.auth.secret_token
+----
+
+[float]
+==== Supported {ecloud} settings
 
-* `output.elasticsearch.flush_bytes`: threshold for bulk request size, in bytes (with suffix), before flushing to {es}. The value must have a suffix, e.g. `"1MB"`, and defaults to `5MB`.
-* `output.elasticsearch.flush_interval`: max duration for accumulating events for a bulk request before it is flushed to {es}. The value must have a duration suffix, e.g. `"5s"`, and defaults to `1s`.
+Below is a list of all **supported settings** (in alphabetical order) for
+users upgrading an {ecloud} standalone (legacy) cluster to {stack} version 8.0.
+Any previously supported settings not listed below will be removed when upgrading.
 
-// talk about no queue
-// see https://github.com/elastic/apm-server/issues/6662 for more
-// jaeger
-// instrumentation
-// source map upload endpoint
-// onboarding docs
+[source,yml]
+----
+apm-server.agent.config.cache.expiration
+apm-server.aggregation.transactions.* //experimental
```

Comment:
@elastic/apm-server Is this setting still experimental? it may have been at some point but it doesn't seem to be now.

Reply:
No, it's not experimental.